api/stripe/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "stripe",
+  "name": "@nikita/stripe",
```

Comment:
This caused issues because you were also trying to install the actual `"stripe"` package